src/config.ts

```diff
@@ -126,6 +126,15 @@ export interface Config {
   background?: string;
 
   /**
+   * The default visualization padding, in pixels, from the edge of the visualization canvas to the data rectangle. This can be a single number or an object with `"top"`, `"left"`, `"right"`, `"bottom"` properties.
+   *
+   * __Default value__: `5`
```

Comment:
Should we just file an issue and deal with this when we make the docs work?
(It's hard to decide this now.. and we should get the alpha.0 out first!)